src/js/modules/articles/reducer.js

```diff
@@ -3,122 +3,88 @@ import {
-      volume: 108,
-      issue: 1,
-      date: "July 19, 2017",
-      time: "8:27 AM",
-      sectionSlug: "creative-writing",
-    },
-  },
+  articles: {},
 };
 
 const reducer = (state = { ...initialState }, action) => {
   switch (action.type) {
     case FETCH_ARTICLE_PENDING: {
-      return { ...state, isFetching: true };
-    }
-    case FETCH_ARTICLE_REJECTED: {
-      return { ...state, isFetching: false, error: action.payload };
+      return { ...state, isFetching: true, };
     }
     case FETCH_ARTICLE_FULFILLED: {
       return {
         ...state,
         isFetching: false,
         isFetched: true,
-        request: action.payload,
+        // action.payload already tested in actions to be array
+        articles: action.payload.reduce((acc, cur) => {
+          const articleSlug = cur.slug;
+          const sectionID = cur.section_id;
+          delete cur.section_id;
+          acc[ articleSlug ] = {
+            ...cur,
+            dateline: formatDate(cur.created_at),
+            sectionSlug: slugFinder(sectionID, action),
+          };
+          return acc;
+        }, { ...state.articles }),
+      };
+    }
+    case FETCH_ARTICLE_REJECTED: {
+      return {
+        ...state,
+        isFetching: false,
+        error: action.payload,
       };
     }
   }
   return state;
 };
 
-export default reducer;
+const formatDate = (string) => {

```

Comment:
Consider factoring out into utils or a selector. You can store ISO and then just use this to display